backend/src/util/tokenChecker.js

```diff
@@ -0,0 +1,20 @@
+const JWT = require("jsonwebtoken");
+
+const tokenChecker = (req, res, next) => {
+  const authHeader = req.headers.authorization;
+  if (authHeader) {
+    const token = authHeader.split(' ')[1];
+    JWT.verify(token, 'SECRET', (err, decoded) => {
```

Comment:
JWT function that takes the user token and our sercret key 'SECRET' and sees if they match.